invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/detail.html

```diff
@@ -5,123 +5,124 @@
 {% macro list_sizes(sizes) %}
   {% for size in sizes %}
     {{ size }}{{ ", " if not loop.last }}
   {% endfor %}
 {% endmacro %}
 
+
 {% macro list_formats(formats) %}
   {% for format in formats %}
     {{ format }}{{ ", " if not loop.last }}
   {% endfor %}
 {% endmacro %}
 
+
 {% macro show_add_titles(add_titles) %}
   {% for add_title in add_titles %}
     {% if add_title.lang %}
-    {{ show_title_detail(add_title.type.title_l10n, add_title.lang.title_l10n, add_title.title)}}
-  {% else %}
-    {{ show_title_detail(add_title.type.title_l10n,None, add_title.title)}}
-  {% endif %}
-{% endfor %}
+      {{ show_title_detail(add_title.type.title_l10n, add_title.lang.title_l10n, add_title.title)}}
+    {% else %}
+      {{ show_title_detail(add_title.type.title_l10n,None, add_title.title)}}
+    {% endif %}
+  {% endfor %}
 {% endmacro %}
 
+
 {% macro show_add_descriptions(add_descriptions) %}
   {% for add_description in add_descriptions %}
-    {{
-      show_sanitized_detail(add_description.type.title_l10n, add_description.description)
-    }}
+    <h3>{{add_description.type.title_l10n }}</h3>
+    <p>{{ add_description.description | sanitize_html() | safe}}</p>
   {% endfor %}
 {% endmacro %}
 
+
 {% macro show_dates(dates) %}
   {% for date in dates %}
-    {{ show_detail(date.type.title_l10n, date.date) }}
+    <dt class="ui tiny header">{{ date.type.title_l10n }}</dt>
+    <dd>
+      <div>{{ date.date }}</div>
+      <div class="text-muted">{{ date.description }}</div>
+    </dd>
   {% endfor %}
 {% endmacro %}
 
+
 {% macro show_funding(funding) %}
   {% for fund in funding %}
     {{ show_detail('Award', fund.award.title + ' ( ' + fund.award.number + ' )' ) if fund.award }}
     {{ show_detail('Funder', fund.funder.name) if fund.funder }}
   {% endfor %}
 {% endmacro %}
 
+
 {% macro show_references(references) %}
+    <dt class="hidden">References</dt>
   {% for reference in references %}
-    <p>{{ reference.reference + ' ( ' + reference.scheme + ' - ' +  reference.identifier + ' )'}}</p>
+    <dd>{{ reference.reference + ' ( ' + reference.scheme + ' - ' +  reference.identifier + ' )'}}</dd>
   {% endfor %}
 {% endmacro %}
 
+
 {% macro _identifiers_for_group(related_identifiers) %}
+  <dt class="hidden">Related identifiers</dt>
   {% for identifier in related_identifiers %}
-    <div class="ui relaxed grid">
-      <div class="two column row">
-        <div class="eight wide column">
-          <p>{{ identifier.identifier + ' ( ' + identifier.scheme | get_scheme_label + ' )' }}</p>
-        </div>
-        <div class="eight wide column">
-          {% if identifier.resource_type is defined %}
-            {{ identifier.resource_type.title_l10n }}
-          {% endif %}
-        </div>
-      </div>
-    </div>
+    <dd>
+      {% if identifier.resource_type is defined %}
+      {{ identifier.resource_type.title_l10n }}:
+      {% endif %}
+
+      {{ identifier.identifier + ' ( ' + identifier.scheme | get_scheme_label + ' )' }}
```

Comment:
Both "Identifiers" and "Related works" sections should make the identifiers linkable. For that we need ``idutils`` library. Let's talk and I'll show you have to them into links